lib/rubocop/config.rb

```diff
@@ -295,6 +296,15 @@ def target_ruby_version
         end
     end
 
+    def target_rails_version
+      @target_rails_version ||=
+        if for_all_cops['TargetRailsVersion']
```

Comment:
`#fetch` should be supported by all Rubies we support. I agree with you `#fetch` is nicer. 🙂